.eslintrc.yaml

```diff
@@ -13,6 +13,7 @@ overrides:
   - files:
       - test/fixtures/**
     rules:
+      jest/require-hook: off
```

Comment:
if this is not disabled for the fixture files, they all generate a message about this rule, like the fixture has to be in a hook, which is not true, so to me it looks like it should be disabled for these files. not sure though if additional configuration for this rule is needed for it to work correctly in other repos